NEWS.md

```diff
@@ -3,6 +3,51 @@ News
 
 As per GCS § 6.7, this file contains a list of user-visible, noteworthy changes. Note that this is not the same as a changelog.
 
+Version 3.2.1 (Development 1)
+-----------------------------
+
+Changes
+
+ * Optimized overall performance related to Soulseek server connections and filesystem sockets
+ * Close redundant peer connections and don't attempt to reconnect to server if logged in elsewhere
+ * Optimized performance and improve robustness of the round robin queue system (thank you @toofar)
+ * Optimized scrolling performance and avoid FPS drops when scrolling large lists containing country flags
+ * Optimized parent row expansions when adding new search results and transfers into tree views
+ * Optimized loading performance of downloads/uploads history and avoid unnecessary saving of transfer lists
+ * Optimized text entry auto-completion performance and reduce the memory usage of open chat tabs
+ * Display real folder path for transfers in the Uploads tab and local items in the File Properties dialog
+ * Improved consistency of translated startup and shutdown logging messages
+ * Added support for toggling dark mode on newer systems using the Adwaita theme
+ * Added support for Python 3.10.1+ (minimum Python 3 version requirement is still 3.5.0)
+ * Improved support for PyGObject GTK 4.4.0+ (minimum GTK 3 version requirement is still 3.18.0)
+ * 
+
+Corrections
+
+ * IMPORTANT (Windows and macOS): Fixed an issue where the system language was not automatically detected
+ * IMPORTANT (Windows and macOS): Fixed an issue where translations were not applied to UI elements
```

Comment:
Perhaps you could combine these two items into one, since translations didn't work on these systems at all (unless you use the env variable, but it's irrelevant to 99% of users).